### PR TITLE
Update ELK to "proper" images

### PR DESCRIPTION
**Pull request by tianon**
~~This is a concrete test of what I was suggesting in https://github.com/docker-library/official-images/issues/15753#issuecomment-1834442877 (not intended for merge as-is)~~

This updates them to point to the original source repository so they build directly instead of being a special case, and updates the maintainers appropriately.

See https://github.com/docker-library/official-images/issues/15753#issuecomment-1834442877 and following conversation :+1:

### CODE REVIEW

**Comment by tianon** [comment]
A GitHub team is OK, but we'll want at least one individual listed too because someone outside the org/team unfortunately can't @-mention a team on GitHub (so neither us nor our bot would be able to do so :disappointed:).

**Comment by mark-vieira** [comment]
> A GitHub team is OK, but we'll want at least one individual listed too because someone outside the org/team unfortunately can't @-mention a team on GitHub (so neither us nor our bot would be able to do so 😞).

Grr. Does the contact have to be a github alias? Could it be an email mailing list or something? Just trying to avoid the obvious problem of folks moving teams, leaving the org, etc.

**Comment by mark-vieira** [comment]
> (Pulling that thought thread a little is probably going to lead to some questions that might be answered by https://github.com/docker-library/faq#can-i-use-a-bot-to-make-my-image-update-prs so I'll just drop that here in case it does 😄)

@alpar-t this might be something to consider going forward.

**Comment by mark-vieira** [comment]
> Email is an optional component of the format in the file, but most of our pings happen on GitHub (like you can see in the bot comment above on this PR) 😞

In the interest of getting our arm images available sooner rather than later I'm good with assigning myself for now. @watson, and @jsvd, thoughts?

**Comment by watson** [comment]
I'm not the correct maintainer for Kibana. I'm currently reaching out internally to find the right person. However, if you want to merge this to unblock the ARM images, go ahead. We can always update this maintainer once I figure out who it is.

**Comment by watson** [comment]
After checking, I think I'm probably the best candidate to put down as the maintainer for Kibana after all 👍

**Comment by jsvd** [comment]
Sign me up. In the future I'd like for the first point of contact to be a mailing list and potentially a bot, to avoid the issues raised by @mark-vieira.

**Review by yosifkit — APPROVED** [review]

**Pull request merged**
